crates/net/Cargo.toml

```
@@ -99,3 +100,5 @@ eventsource = [
     'web-sys/EventSource',
     'web-sys/MessageEvent',
 ]
+# As of now, only implements `AsyncRead` and `AsyncWrite` on `WebSocket`
+io-util = ["futures-io"]
```

```suggestion
io-util = ["dep:futures-io"]
```
This ensures `futures-io` is not a feature (by default, cargo adds features for optional dependencies)

---

I agree with you. However, the rest of the Cargo.toml is currently not using `dep:` to denote a dependency. When using this notation at one place, [the other places must be updated too because this effectively disables the old "implicit features"](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies). I think it’s better to tackle this in a separate PR for all crates in order to keep consistency. However, if you prefer I’m okay with converting this crate as part of this PR. I just want to make sure it’s okay for you.